service/src/test/java/bio/terra/pipelines/stairway/imputation/AddWdsRowsStepTest.java

```diff
@@ -30,7 +30,13 @@ class AddWdsRowsStepTest extends BaseEmbeddedDbTest {
   void setup() {
     FlightMap inputParameters = new FlightMap();
     FlightMap workingMap = new FlightMap();
+
+    inputParameters.put(
+        RunImputationJobFlightMapKeys.PIPELINE_INPUT_DEFINITIONS, "i'm not the right type");
```

Comment:
is there a reason we're putting an incorrect type here?

Reply:
whoops, this was leftover from something I was testing, nice catch